packages/backend-common/src/service/lib/ServiceBuilderImpl.ts

```diff
@@ -33,7 +33,7 @@ import { ServiceBuilder } from '../types';
 import { readBaseOptions, readCorsOptions } from './config';
 
 const DEFAULT_PORT = 7000;
-const DEFAULT_HOST = 'localhost';
+const DEFAULT_HOST = '';
```

Comment:
I think `undefined` would be the most explicit default tbh, signaling that we'll not pass on anything to express/NodeJS

Reply:
testing

Reply:
I have to do this to make typescript happy:

```js
      const server = host ? 
        app.listen(port, host, () => {
          logger.info(`Listening on ${host}:${port}`);
        }) :
        app.listen(port, () => {
          logger.info(`Listening on :${port}`);
        });

      const stoppableServer = stoppable(server, 0);
```

kind of messy,  but do you prefer that?

Reply:
Ah meh, alright should go with `''` then